ChangeLog.md

```diff
@@ -4,29 +4,244 @@ Notable changes to the FreeOrion project will be documented in this file.
+    - Added random species option to species droplists when setting up games.
+    - Clicking the random seed button on the galaxy setup screen now sets the seed to "Random" which will be replaced by the server with a randomly-generated seed.
+    - Various values in GUI should now round to nearest rather than next lower value for their displayed precision.
+    - Added lists of available parts, hulls, and buildings to empire pedia articles.
+    - Sorted researched techs list on empire pedia article by researched turn.
+    - Made several GUI widgets react immediately to being pressed, instead of waiting until the mouse is released to complete the "click".
 
 #### Content
 
+- New Pedia articles, article categorization, extended articles, corrections
+- Translation updates: French, Russian
+- Added dimensional matrix engine part.
+- Made additional species playable.
+- Added Replicon species.
+- Tweaked various default ship designs.
+- Replaced multiple fuel parts with upgrades to a single part.
+- Added Next Eradicator building.
+- Added support for pedia articles for planets having different landscape images per-planet.
+- Added population statistic.
+- Added a sitrep when a ship naturally goes from 0.x to 1.x fuel.
+- Added the Flux Bubble hull.
+- Game rules:
+    - Enable random reseeding
+    - Stockpile import limits
+    - Ship, buildings, techs are 1 PP/RP and 1 turn to produce / research
+    - Number of rounds of combat
+    - Enable experimentors
+    - Enable super test takeover building
+    - Enable conceding and max allowed colonies when conceding from a multiplayer game
+    - Restrictions on allied victories
+    - Scale ship speed, ship structure, building cost, tech cost, hull cost, part cost
+    - Planet size balance adjustments
+    - How much to over-allocate on production queue to prevent rounding-related extra turns to complete items
 
 #### Balance
 
+- Super testers now have perfect stealth, which allows testing without AI interference.
+- Restricted detector, stealth, and shield parts to one per ship.
+- Adjusted various species' balance.
+- Made scrying sphere set visibility to the greater of current and partial visibility.
+- Made telepathic detection set visibility to the greater of current and basic visibility.
+- Reworked resupply / upgrading of ship parts, based on the last turn a ship was resupplied and the turn a tech was researched.
+    - Upgrades now don't happen during the pre-combat movement phase of turns.
+- Reduced cost of Transcendent Design tech.
+- Required gifted fleets to be stationary.
+- Ancient guardians self-destruction only triggers on planets, rather than all possible objects with species.
+- Gas Giant Generator now gives a smaller bonus if the planet is populated.
+- Reduced Space Elevator bonus for Gas Giants by 1.
+- Increased Colony Base hull cost to 3.
+- Newly created ships and monsters now cannot block supply.
```

Comment:
add "or enemy fleet movement" to end of sentence
and perhaps also add ", until they have survived a turn" to after that

Reply:
Incorporated first suggestion, and put the second one after "...cannot block supply" in parenthesis, as that only applies in this case. AFAIK newly created monsters can't block movements of enemy fleets that have already been present in the system, even if they survive at least one turn of combat.